### PR TITLE
Fix #15036 [com_tags] Undefined index: tag

### LINKED ISSUE #15036: [com_tags] Undefined index: tag
### Steps to reproduce the issue
Go to Components > Tags
Click `Check All` checkbox.
Click `Batch` button.
Click `Process` button.

In PHP error log:
> PHP Notice:  Undefined index: tag in \joomla-cms-staging\libraries\legacy\model\admin.php on line 258

### PR DESCRIPTION
**Pull request by ghost**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/15036

### Summary of Changes
Added model specific `$batch_commands` array (removed index `tag` from array).

### Testing Instructions
See https://github.com/joomla/joomla-cms/issues/15036

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 4eb371bd6fc92d06806f6300f47e7c57c9689cd3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15040">issues.joomla.org/tracker/joomla-cms/15040</a>.</sub>

File: administrator/components/com_tags/models/tag.php [hunk 1]
```diff
@@ -32,6 +32,17 @@ class TagsModelTag extends JModelAdmin
 	public $typeAlias = 'com_tags.tag';
 
 	/**
+	 * Allowed batch commands
+	 *
+	 * @var    array
+	 * @since  3.4
```
Comment (brianteeman) [inline administrator/components/com_tags/models/tag.php]: is it not "@since 3.7 ??

Comment (zero-24) [inline administrator/components/com_tags/models/tag.php]: fixed @brianteeman cc: @bertmert

Comment (bertmert) [inline administrator/components/com_tags/models/tag.php]: Thank you. I'll never understand when what version ;-)

Comment (zero-24) [inline administrator/components/com_tags/models/tag.php]: np just ping me and i try to help you 😄

**Comment by AlexRed** [comment]
I have tested this item :white_check_mark: successfully on cd1c8e5dc475a37ab21706b88d690686045ee0d1<br /><br />Patch ok for me<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15040">issues.joomla.org/tracker/joomla-cms/15040</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on cd1c8e5dc475a37ab21706b88d690686045ee0d1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15040">issues.joomla.org/tracker/joomla-cms/15040</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful testes.

**Pull request merged**